pkg/cache/clusterqueue_snapshot.go

```diff
@@ -149,3 +149,17 @@ func (c *ClusterQueueSnapshot) DominantResourceShareWithout(wlReq resources.Flav
 	}
 	return dominantResourceShare(c, without)
 }
+
+func (c *ClusterQueueSnapshot) FindTopologyAssignments(
```

Comment:
Can we maybe rename it to `FindTopologyAssignmentsForWorkload` or something similar to avoid repeating the name of the method?

Reply:
Done, and I wrapped the scary map[Flavor][]TASPodSetRequests into WorkloadTASRequests to encapsulate the idea,